### PR TITLE
Ibn labs/additional module directories dev 4

### PR DESCRIPTION
**Pull request by qt1**
A variant of PR 5604 - against dev

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.DynamicForms/ValidationRules/Compare.cs [hunk 1]
```diff
@@ -22,7 +21,9 @@ public class Compare : ValidationRule {
         }
 
         private LocalizedString GetValidationMessage(ValidationContext context) {
-            return new LocalizedString(Tokenize(T(ErrorMessage.WithDefault(String.Format("{0} must match the value of {1}.", context.FieldName, TargetName))).ToString(), context));
+            return String.IsNullOrWhiteSpace(ErrorMessage)
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/ValidationRules/Compare.cs]: This has nothing to do with the PR

**Comment by sebastienros** [comment]
Could you try to squash your changes in a single commit, and "force push" to the same branch you are currently working on? This PR will be updated automatically. If you don't know how to do, just ping me on skype or gitter.

**Comment by qt1** [comment]
#5982 is a squashed variant relative to 1.9.x .

**Comment by sebastienros** [comment]
Almost !

**Pull request closed without merging**